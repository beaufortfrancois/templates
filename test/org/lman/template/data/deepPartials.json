{
  "one": {
    "foo": 1,
    "bar": 2
  },
  "two": {
    "foo": 3,
    "bar": 4
  }
}
