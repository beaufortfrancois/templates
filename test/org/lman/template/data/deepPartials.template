a-{{#one}}{{+templates.deepPartials_partial}}{{/}}
b-{{#two}}{{+templates.deepPartials_partial}}{{/}}
