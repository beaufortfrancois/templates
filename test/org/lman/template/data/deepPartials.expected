a-x1y2z

b-x3y4z

